skywalking/config.py

```diff
@@ -86,6 +86,11 @@
                       '%(asctime)s [%(threadName)s] %(levelname)s %(name)s - %(message)s'  # type: str
 cause_exception_depth = int(os.getenv('SW_AGENT_CAUSE_EXCEPTION_DEPTH') or '5')  # type: int
 
+mysqlclient_host = os.getenv('SW_MYSQLCLIENT_HOST') or '127.0.0.1'
+mysqlclient_port = os.getenv('SW_MYSQLCLIENT_PORT') or '3306'
+mysqlclient_db = os.getenv('SW_MYSQLCLIENT_DB') or ''
```

Comment:
These should not be configured, they should be detected automatically.

Reply:
ok，we edit it.

Reply:
These should be removed now